go/stream/message_test.go

```diff
@@ -418,6 +556,10 @@ func TestMessage(t *testing.T) {
 			logger.NewStringField("compression", "some-serialized-compression-settings"),
 			logger.NewStringField("encryption", "some-serialized-encryption-settings"),
 			logger.NewStringField("encryptionKeyID", "encryptionKeyID"),
+			logger.NewBoolField("isBot", false),
+			logger.NewStringField("botName", ""),
+			logger.NewStringField("botURL", ""),
+			logger.NewBoolField("botUnknownBrowser", false),
```

Comment:
Why do we include these fields at all if `isBot` is `false`? We should add data only if needed and try to be more responsible for the data we carry around with us.